CoreFoundation/StringEncodings.subproj/CFStringEncodingConverter.h

```diff
@@ -72,16 +72,16 @@ CF_EXPORT uint32_t CFStringEncodingBytesToUnicode(uint32_t encoding, uint32_t fl
 typedef CFIndex (*CFStringEncodingToBytesFallbackProc)(const UniChar *characters, CFIndex numChars, uint8_t *bytes, CFIndex maxByteLen, CFIndex *usedByteLen);
 typedef CFIndex (*CFStringEncodingToUnicodeFallbackProc)(const uint8_t *bytes, CFIndex numBytes, UniChar *characters, CFIndex maxCharLen, CFIndex *usedCharLen);
 
-CF_EXPORT bool CFStringEncodingIsValidEncoding(uint32_t encoding);
+CF_PRIVATE bool CFStringEncodingIsValidEncoding(uint32_t encoding);
```

Comment:
`CF_PRIVATE` is not defined for people importing CF's headers, so if something is appearing in a public or private header that is not actually exported, we need to just delete it.

Reply:
Ah, cool, will remove.  I was just worried about doing that since it was previously in the header.  I'll remove it.

Reply:
Hmm, I actually took another look at this.  This header is private and will not be exposed to the users, so I think that we can leave this as is.  Do you still want to move it to another header?  The implementation and the use are across two files, so we need it to be in some header.

Reply:
On Darwin (where this source code is shared), this header is used by other projects outside CF, so if we change this to CF_PRIVATE then we will break them. We call that "private" vs an "internal" header. Unfortunately there is not really a way for you to see that from this project.

Reply:
It doesn't help that `CF_PRIVATE` is a misnomer; it really should be `CF_INTERNAL`, but anyway that's something we shouldn't try to change here.

Reply:
Okay, I can move the declaration, but do need a new home for it.

Reply:
`CFStringEncodingConverterPriv.h`, despite the name looking like it's private, is actually internal. That is a good place.